README.md

```diff
@@ -53,9 +53,14 @@ This documentation provides a comprehensive overview of the UKEF Digital TradeFi
 6. Set UKEF TFM environment variables in your terminal: `UKEF_TFM_API_SYSTEM_KEY` and `UKEF_TFM_API_REPORTS_KEY`.
 7. Run `npm run env:copy` to copy your root .env file into all the individual projects that need it.
 8. Run `npm ci` in the root folder of the repository. (note: this will install dependencies for the entire project, including those specified in sub-packages. More details on this in the [npm workspaces](./doc/npm-workspaces.md) docs)
-9. Start your local environment with `npm run start`.
-10. Run migrations on the MSSQL Server database (see [SQL DB docs](./doc/sql-db.md#--run-migrations) for details)
-11. Create mock data by running `npm run load` from the root folder of the repository. This should generate mocks in your database (both Mongo and MSSQL). (for more details on what this does please see [utils docs](./utils/README.md))
+9. Start your local environment with `docker-compose up --build`.
+10. Create mock data
+    - If not working on utilisation reports:
+      1. Run migrations on the MSSQL Server database by running `npm run db:migrate` in the `libs/common` directory (see [SQL DB docs](./doc/sql-db.md#--run-migrations) for details)
+      2. Create mock data by running `npm run load` from the root folder of the repository. This should generate mocks in your database (both Mongo and MSSQL). (for more details on what this does please see [utils docs](./utils/README.md)).
+    - If working on utilisation reports or E2E tests for utilisation reports:
+      1. Run the command `npm run load:init` from the root folder of the repository.
```

Comment:
This all reads to me that you need to run the following commands. Is this correct?

```
npm run load
npm run db:migrate
npm run load:init
```

Reply:
nope.  it is either 
npm run db:migrate and then npm run load if not utilisation report work
npm run load:init if utilisation report work

Reply:
Why do we need two different command for utilitsation report, can they not be one `npm run load:report` which then executes db-migration and other scripts.